static/app/views/organizationGroupDetails/groupMerged/index.tsx

```diff
@@ -73,11 +75,18 @@ class GroupMergedView extends React.Component<Props, State> {
   listener = GroupingStore.listen(this.onGroupingChange, undefined);
 
   getEndpoint() {
-    const {params} = this.props;
+    const {params, organization, location} = this.props;
     const {groupId} = params;
 
+    const hasGroupingTreeFeature = organization?.features?.includes('grouping-tree-ui');
```

Comment:
can `Organization` be `undefined`?

Reply:
Yes, apparently. I am forced to make this the type.